templates/config.yaml.j2

```diff
@@ -36,6 +36,9 @@ node-label:
 {% endif %}
 snapshotter: {{ rke2_snapshooter }}
 node-name: {{ inventory_hostname }}
+{% if ( disable_kube_proxy is defined) %}
```

Comment:
I think it would be better to evaluate if the variable is `true` or `false`. Maybe you could use `sameas` for this.

Reply:
Sure, that should be fine too.

Reply:
done